src/sentry/static/sentry/app/views/dashboardsV2/widget/new/utils.tsx

```diff
@@ -11,3 +13,48 @@ export const metricMockOptions = [
   'sentry.events.processed.node',
   'symbolicator.healthcheck',
 ];
+
+export const metricTags = {
```

Comment:
Is this constrained to begin with, or will this grow? Some of the values seem like they might not work for all customers.

Reply:
this is just a mock...the API is not yet implemented... as soon as I am done with events I will put everything related to metrics under  a feature flag.